identity/webapp/src/utility/auth0.ts

```diff
@@ -1,7 +1,7 @@
 import {
   initAuth0,
-  WithPageAuthRequiredOptions,
   PageRoute,
+  WithPageAuthRequiredPageRouterOptions,
```

Comment:
I _think_ this is right but I weirdly couldn't find anything about `WithPageAuthRequiredOptions` in the documentation. Although I can't find exactly where the change happened, this type seems to be the new one to use as it contains `returnTo`.